hack/lib/lint.sh

```diff
@@ -18,11 +18,11 @@
 
 set -o errexit
 set -o nounset
-set -o pipefail
 
 kubeedge::lint::check() {
     cd ${KUBEEDGE_ROOT}
     # skip deleted files
+    set +o pipefail
```

Comment:
Just curious, why this doesn't prevent the pr https://github.com/kubeedge/kubeedge/pull/1813 from merging?

Reply:
Since it compares with master branch, but after merge, there is no different files, then grep nothing and return 1.